go.sum

```diff
@@ -207,6 +206,8 @@ github.com/google/go-github v17.0.0+incompatible h1:N0LgJ1j65A7kfXrZnUDaYCs/Sf4r
 github.com/google/go-github v17.0.0+incompatible/go.mod h1:zLgOLi98H3fifZn+44m+umXrS52loVEgC2AApnigrVQ=
 github.com/google/go-github/v20 v20.0.0 h1:Ffvfyn3+lnxI1BtP4xcxV0S6TrCh6L1IUv5bN4GWsNA=
 github.com/google/go-github/v20 v20.0.0/go.mod h1:IRYtAIrijh8T1S+AlIsVlBt3/GT4sAT110wzAVCzevM=
+github.com/google/go-github/v27 v27.0.0 h1:8fNZ10VRSgTer3Nz/FwQpawaTSQaBSKLoJhFw078pOE=
```

Comment:
This is strange, we shouldn't have dependencies on github.com/google/go-github/v20 and github.com/google/go-github/v27. Do a search and find out where github.com/google/go-github/v27 is imported; if it's not imported anywhere, delete this.